### DISCUSSION TITLE
Starting with 0.8.0 series google closure compiler users cannot use this library

### DISCUSSION THREAD

**Discussion by geraldodev** [KevinVandy/material-react-table#328] [General]
```
Errors encountered while trying to parse file
  /home/geraldo/projetos/t3-ti/node_modules/@tanstack/match-sorter-utils/build/lib/remove-accents.js
  {:line 131, :column 3, :message "Character '̆' (U+0306) is not a valid identifier start char"}
```
It's valid javascript but not recognized by google closure compiler, the one who backs the clojurescript compiler.

https://github.com/google/closure-compiler/issues/3639

match-sorter-utils replaced match-sorter which didn't not have this closure compiler issue.

I've had the same problem with tanstack react-query devtools

Could you please use match-sorter and unlock this awesome library to closure compiler users ?

Regards,

Geraldo Lopes de Souza

**Comment by KevinVandy** [comment]
I don't know what that is, but no, TanStack match sorter utils must be used here, as only it exposes the the functionality needed internally. There was a recent update a month ago in that package that made it compatible with older versions of webpack though, so just upgrade to the latest version of MRT

**Comment by geraldodev** [comment]
It's valid javascript the utf written in tanstack, but google closure compiler does not recognize it, due to internal error.
the difference between the two is that tanstack version save some bytes with the literals being used.
As Thomas Heller told me "the closure compiler doesn't like js objects with unicode keys such as Ã: 'A',
"Ã": 'A', would be fine"

The match-sorter-utils utilizes " so a couple of bytes more.


Good luck with the library

**Reply by KevinVandy** [reply]
That issue could be fixed upstream in that library. Make a PR or report it there
